pypiserver/Tiltfile

```diff
@@ -0,0 +1,66 @@
+EXT_PATH = os.getcwd()
+PYPI_PORT_VAR_NAME = 'PYPISERVER_PORT' # docker compose purposes
+DEFAULT_PYPI_PORT_VALUE = '8222'
+DEFAULT_CACHE_DIR = '.tilt-cache'
```

Comment:
i'm not sure it makes much sense to call this a `tilt cache`. or use `TILT_CACHE_DIR` to configure it

Reply:
so `.tilt-cache` is just a fallback, default value when there is no TILT_CACHE_DIR